contracts/Nexus.sol

```diff
@@ -102,8 +102,8 @@ contract Nexus is INexus, BaseAccount, ExecutionHelper, ModuleManager, UUPSUpgra
     {
         address validator = op.nonce.getValidator();
         if (!op.nonce.isModuleEnableMode()) {
-            // Check if validator is not enabled. If not, return VALIDATION_FAILED.
-            if (!_isValidatorInstalled(validator)) return VALIDATION_FAILED;
+            // Check if validator is not enabled. If not, revert.
+            require(_isValidatorInstalled(validator), InvalidModule(validator));
```

Comment:
this should then also apply to both `if` and `else` branches in https://github.com/bcnmy/nexus/pull/139/files as both branches there return this constant